crates/sui-storage/src/indexes.rs

```diff
@@ -148,6 +166,18 @@ fn dynamic_field_index_table_default_config() -> DBOptions {
 fn index_table_default_config() -> DBOptions {
     default_db_options()
 }
+fn coin_index_table_default_config() -> DBOptions {
+    DBOptions {
+        options: optimized_for_high_throughput_options(
```

Comment:
not related to this PR, but based on your investigation for val-03(as well as data from val-02), the rate of compactions is very high and we might want to tune this options a bit to reduce compactions.

`optimized_for_high_throughput_options` is optimizing for high write throughput, in the case of the coin index table, intuitively we should pick a setting that's optimized for read throughput?

Reply:
yeah I think there are optimizations to be done. If we look at the nodes with this index, cpu is mostly spent on rocks iterator seeking and that kind of stuff, this option does not optimize for that